Source/Scene/LabelCollection.js

```diff
@@ -407,7 +407,7 @@ function repositionAllGlyphs(label) {
     horizontalOrigin,
     backgroundPadding
   );
-  var lineSpacing = defaultLineSpacingPercent * maxLineHeight;
```

Comment:
I question why line-spacing is based upon the maximum glyph height instead of the font-size.

Reply:
Not quite sure either.  Your calculation looks fine, but we just need to make sure letters than go below the base like like 'y' and 'g' don't overrun tall letters like 'T' on the line below it

Reply:
The existing logic causes multi-line label "heights" to only be as tall as the tallest character in the string, instead of a fixed height based upon the font-size (which is how I understand CSS `font-size` to work).

So label with text `_\n_\n_\n_` renders with effectively small "line-heights" compared to one with text `M\nM\nM\nM`.
![image](https://user-images.githubusercontent.com/8540239/84547391-117c8a80-acc9-11ea-9e49-7d3b6f8c9be8.png)